client/lib/site/index.js

```diff
@@ -86,46 +87,7 @@ Site.prototype.set = function( attributes ) {
  * Update any computed attributes
  */
 Site.prototype.updateComputedAttributes = function() {
-	/**
```

Comment:
> Still think this is yet to be addressed https://github.com/Automattic/wp-calypso/pull/6477/files#r73341185

This links to:
> I think there are a handful of unused dependencies after removing this block of logic. I've not checked out the branch yet, but at a glance at least trim and config.

Which I addressed [here](https://github.com/Automattic/wp-calypso/pull/6477/files#r73387334) and [here](https://github.com/Automattic/wp-calypso/pull/6477/files#r69311609) ;)